assets/app/styles/_components.less

```diff
@@ -169,6 +169,10 @@
     border-radius: @border-radius-md;
     max-width: 600px;
     padding: 5px 10px;
+    // so that .word-break() on children works
+    [flex] {
+      min-width: 0;
```

Comment:
but only within the pod template, just double check all the browsers and im ok with it